app/views/taxons/show.html.erb

```diff
@@ -15,7 +15,10 @@
     <div class="taxon-page__section-group">
       <div class="grid-row">
         <div class="column-two-thirds">
-          <h2 class="taxon-page__section-heading"><%= section[:title] %></h2>
```

Comment:
I wondered if we could also remove the CSS definition for `.taxon-page__section-heading` but it's also being used in `app/views/taxons/_organisations.html.erb`. Perhaps we could update that heading too and then remove the CSS?

Reply:
Good spot, thanks!

Reply:
@andrewgarner done 🙂